cli/src/plugins/javadoc/index.ts

```diff
@@ -201,15 +214,17 @@ function makeInheritedMethodList(args: MakeInheritedMethodListArgs) {
     md.list(
       "unordered",
       args.list.map((interfaceType) => {
-      return [
-        md.listItem([md.text(args.prefix + interfaceType.qualifiedTypeName),
-        md.list(
-          "unordered",
-          args.inheritedMethods[interfaceType.qualifiedTypeName].map((method) => {
-            return md.listItem(args.project.linkToEntity(interfaceType.qualifiedTypeName, method))
-          }))])
-      ]
-    }).flat(1))
+        return [
+          md.listItem(
+            [
+              md.text(args.prefix), args.project.linkToEntity(interfaceType.qualifiedTypeName), md.text(": "), md.paragraph(),
```

Comment:
Are you absolutely sure you're running prettier?

Reply:
It is installed, enabled, configured to run on save, and set as the default formatter. no clue why it isn't formatting on save